helix-term/src/ui/statusline.rs

```diff
@@ -407,44 +409,41 @@ where
     write(context, format!(" {} ", file_type), None);
 }
 
-fn render_file_name<F>(context: &mut RenderContext, write: F)
+fn render_title<F>(context: &mut RenderContext, write: F, path: Option<PathBuf>)
```

Comment:
"title" doesn't feel like the right word here since this function is about rendering paths. Maybe `render_file_name_impl` (`*_impl` is already a common pattern in the codebase) or `render_path_name` or just `render_path`?

Reply:
The second commit actually changes it from `render_path `to `render_title`. I’ll be able to revert it once I can access my system. But until then, I’ll try to think of a better name, as the function isn’t just rendering the path but also the modification status of the buffer.